app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,19 @@
+class CommentsController < ApplicationController
+  def create
+    @post = Post.find(params[:post_id])
+    @comment = @post.comments.build(comment_params)
+    @comment.user = current_user
+
+    if @comment.save
+      redirect_to @post, notice: 'Comment added!'
+    else
+      render 'posts/show'
+    end
+  end
+
```

Comment:
- [ ] Also kindly include the new method here to direct to a form where comment can be created from